src/nvim/testdir/test_filetype.vim

```diff
@@ -1962,4 +1962,35 @@ func Test_inc_file()
   filetype off
 endfunc
 
+func Test_lsl_file()
+  filetype on
+
+  call writefile(['looks like Linden Scripting Language'], 'Xfile.lsl', 'D')
```

Comment:
this requires defer

Reply:
D'oh! We _just_ talked about that, too...

Can we use it without the `D`?

Reply:
yes but you'll need to `:call delete('Xfile.lsl')` manually.

that'll likely make this a partial port :thinking: (until there's consensus on what's happening in regards to porting defer)